### ISSUE TITLE
Cannot initialize service due to parsing error

### ISSUE THREAD

**Issue by pacarole** [tjanczuk/git-azure#57]
I'm trying to initialize a service, but for some reason it cannot parse my *.publishSettings file:

Unable to parse *.publishSettings file from /Users/pascal/Downloads/MSDN-11-14-2012-credentials.publishsettings

I have configured a username, password, and am using a unique service name.

**Comment by limajs** [comment]
I'm getting the same error - I'm sure I've followed the correct steps (as per the video)

**Comment by tjanczuk** [comment]
The issue is related to a loose version binding to xml2js combined with an apparent breaking change in xml2js v0.2.x. I fixed it in 2293d96e35e5fdb53ff0358c39740d8200acf93d by pinning xml2js version to 0.1.13. 

I also released new git-azure v0.6.0 with this fix to npm. To pick it up:

```
sudo npm uninstall git-azure -g
sudo npm install git-azure -g
```

**Comment by pacarole** [comment]
Brilliant, thanks!
